tests/clone_test.go

```diff
@@ -13,8 +13,8 @@ import (
 	"kubevirt.io/kubevirt/pkg/libvmi"
 	"kubevirt.io/kubevirt/pkg/pointer"
 
```

Comment:
nit: Move to block above

Reply:
Done in this [revision](https://github.com/kubevirt/kubevirt/compare/eed4dfd64c3d0d8a8b9fff48933f75ff830f8959..3aa64ad2b9e147718c1376d3bede0ee7eb643671).